build.sh

```diff
@@ -0,0 +1,122 @@
+#!/bin/bash
+
+#Copyright (c) 2019, NVIDIA CORPORATION.
+
+# custrings build script
+
+# This script is used to build the component(s) in this repo from
+# source, and can be called with various options to customize the
+# build as needed (see the help output for details)
+
+# Abort script on first error
+set -e
+
+NUMARGS=$#
+ARGS=$*
+
+ # Get root of git repository without assuming location of build.sh script
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+cd "$SCRIPT_DIR"
+REPODIR=$(git rev-parse --show-toplevel)
+
+VALIDARGS="clean libcustrings custrings -v -g -n -h"
+HELP="$0 [clean] [libcustrings] [custrings] [-v] [-g] [-n] [-h]
+   clean        - remove all existing build artifacts and configuration (start over)
+   libcustrings - build the custrings C++ code only
+   custrings    - build the custrings Python package
+   -v           - verbose build mode
+   -g           - build for debug
+   -n           - no install step
+   -h           - print this text
+    default action (no args) is to build and install 'libcustrings' then 'custrings' targets
+"
+LIBCUSTRINGS_BUILD_DIR=${REPODIR}/cpp/build
+CUSTRINGS_BUILD_DIR=${REPODIR}/python
+BUILD_DIRS="${LIBCUSTRINGS_BUILD_DIR}"
+
+ # Set defaults for vars modified by flags to this script
+VERBOSE=""
+BUILD_TYPE=Release
+INSTALL_TARGET=install
+
+ # Set defaults for vars that may not have been defined externally
+#  FIXME: if INSTALL_PREFIX is not set, check PREFIX, then check
+#         CONDA_PREFIX, but there is no fallback from there!
+INSTALL_PREFIX=${INSTALL_PREFIX:=${PREFIX:=${CONDA_PREFIX}}}
+PARALLEL_LEVEL=${PARALLEL_LEVEL:=""}
+PYTHON=${PYTHON:-python}
+
+ function hasArg {
+    (( NUMARGS != 0 )) && (echo " ${ARGS} " | grep -q " $1 ")
+}
+
+function createBuildArea {
```

Comment:
Remove this function and inline it at the one place it will be called (see below)